bitrise.yml

```diff
@@ -37,10 +38,51 @@ workflows:
     - errcheck:
     - go-test:
     after_run:
+    - test
+
+  test:
+    steps:
+    - script:
+        run_if: $.IsCI
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
```

Comment:
we could create a special script step for this scenario 🤔

Reply:
Should I create a story for this?

Reply:
No I think this is ok